### PR TITLE
Add Deno

### LINKED ISSUE #7: 🖼️ Framework Request: Deno
| Question                                   | ✅ Yes | ❌ No |
|--------------------------------------------|-------|------|
| Is this framework already being requested? | - [ ]    | - [ x ]   |

### What is the name of the framework?

Deno

### What language does the framework use?

TypeScript or JavaScript

### What is the website for the framework?

deno.land

### PR DESCRIPTION
**Pull request by cjdenio**
This PR closes #7.﻿

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,5 @@
 
 ## 🌎 Languages
 
-- 🎯 Dart
+- 🎯 [Dart](languages/dart/README.md)
+- 🦕 [Deno](languages/deno/README.md)
```
Comment (rishiosaur) [inline README.md]: Add links to the folders, not the READMEs.

Comment (cjdenio) [reply README.md]: I actually removed the links entirely, for addition in another PR.

Comment (rishiosaur) [reply README.md]: Oh, ok

**Comment by rishiosaur** [comment]
I think it LGTM. Feel free to go ahead if you want, @cjdenio. You can also wait for @Matt-Gleich if you want, but I think he's AFK.

**Pull request merged**